src/Solvers/Random.jl

```diff
@@ -1,8 +1,64 @@
+using OptimizedEinsum: removedsize, removedsize_noise, ssa_path_cost, ContractionPath
+using OptimizedEinsum.Solvers: greedy_choose_simple!, greedy_choose_thermal!
+using Base: @kwdef
+using Random: seed!
+
 abstract type RandomSolver <: Solver end
 
-struct RandomGreedy <: RandomSolver
-    temperature::Float32
-    rel_temperature::Bool
-    nbranch::Int
-    cost_fn::Core.Function
+@kwdef struct RandomGreedy <: RandomSolver
+    temperature::Float32 = 1.
+    rel_temperature::Bool = true
+    nbranch::Int = 8
+    max_time::Number = 100
```

Comment:
`Number` is an abstract type. By general rule, don't put abstract types on structs. Or do you want it to be a `Float` also?

```suggestion
    max_time::Int = 100
```

Reply:
Yes, I had in mind that maybe somebody might want a `Float` value `max_time`. But since we deleted the part that stops the trial evaluation if `max_time` is reached, we might want to directly delete `max_time`.